src/fparser/tests/fparser2/test_Fortran2003.py

```diff
@@ -3215,53 +3340,105 @@ def test_Module_Subprogram_Part(): # R1107
     '''))
     assert isinstance(a, cls),repr(a)
     assert_equal(str(a),'CONTAINS\nSUBROUTINE foo(a)\n  REAL :: a\n  a = 1.0\nEND SUBROUTINE foo')
-    
-def test_Use_Stmt(): # R1109
 
-    cls = Use_Stmt
-    a = cls('use a')
-    assert isinstance(a, cls),repr(a)
-    assert_equal(str(a),'USE :: a')
-    assert_equal(repr(a),"Use_Stmt(None, Name('a'), '', None)")
 
-    a = cls('use a, only: b')
-    assert isinstance(a, cls), repr(a)
-    assert str(a) == 'USE :: a, ONLY: b'
-    assert repr(a) == "Use_Stmt(None, Name('a'), ', ONLY:', Name('b'))"
+def test_Use_Stmt():  # pylint: disable=invalid-name
+    ''' Tests that USE statement is parsed correctly (R1109) '''
+    ucls = Use_Stmt
+    inst = ucls('use a')
+    assert isinstance(inst, ucls), repr(inst)
+    assert str(inst) == 'USE a'
+    assert repr(inst) == "Use_Stmt(None, None, Name('a'), '', None)"
 
-    a = cls('use a, only : b')
-    assert isinstance(a, cls), repr(a)
-    assert str(a) == 'USE :: a, ONLY: b'
-    assert repr(a) == "Use_Stmt(None, Name('a'), ', ONLY:', Name('b'))"
+    inst = ucls('use :: a')
+    assert isinstance(inst, ucls), repr(inst)
+    assert str(inst) == 'USE :: a'
+    assert repr(inst) == "Use_Stmt(None, '::', Name('a'), '', None)"
 
-    a = cls('use a, ONLY : b')
-    assert isinstance(a, cls), repr(a)
-    assert str(a) == 'USE :: a, ONLY: b'
-    assert repr(a) == "Use_Stmt(None, Name('a'), ', ONLY:', Name('b'))"
+    inst = ucls('use a, only: b')
+    assert isinstance(inst, ucls), repr(inst)
+    assert str(inst) == 'USE a, ONLY: b'
+    assert repr(inst) == (
+        "Use_Stmt(None, None, Name('a'), ', ONLY:', Name('b'))")
 
-    a = cls('use :: a, c=>d')
-    assert isinstance(a, cls),repr(a)
-    assert_equal(str(a),'USE :: a, c => d')
+    inst = ucls('use a, only : b')
```

Comment:
This bit of the test appears to be a duplicate of lines 3358-3362 above.

Reply:
Deleted.